validator/src/journal/chain_commit_state.rs

```diff
@@ -163,17 +160,12 @@ impl TransactionCommitCache {
     }
 
     pub fn contains(&self, transaction_id: &str) -> bool {
-        self.committed.contains(transaction_id) || self.blockstore_has_txn(transaction_id)
-    }
-
-    fn blockstore_has_txn(&self, transaction_id: &str) -> bool {
-        let gil = cpython::Python::acquire_gil();
-        let py = gil.python();
-        self.transaction_committed
-            .call(py, (transaction_id,), None)
-            .expect("Call to determine if transaction is committed failed")
-            .extract::<bool>(py)
-            .unwrap()
+        // Shouldn't expect here
```

Comment:
Can you clarify this comment?

Reply:
This was actually intended as a comment to myself (and I forgot to do anything about it before marking the PR as ready for review). I'm not sure how we should handle these results. It looks like these calls would fail due to a database error; are these errors recoverable? If they are, it's unclear how they should be handled by the publisher.